src/main/java/com/google/cloud/genomics/dataflow/utils/GCSOptions.java

```diff
@@ -29,7 +29,7 @@
 import com.google.cloud.genomics.utils.GenomicsFactory;
 import com.google.common.collect.ImmutableList;
 
-import org.codehaus.jackson.annotate.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
I think you're asking whether some of the imports are unused? As far as I
can tell, all the ones here are in use.

If you're asking whether we can get rid of the GenomicsFactoryFactory then
I don't know, I haven't looked closely at this file yet.

On Thu, Apr 16, 2015 at 3:42 PM, Nicole Deflaux <notifications@github.com>
wrote:

> In src/main/java/com/google/cloud/genomics/dataflow/utils/GCSOptions.java
> <https://github.com/googlegenomics/dataflow-java/pull/55#discussion_r28556833>
> :
>
> > @@ -29,7 +29,7 @@
> >  import com.google.cloud.genomics.utils.GenomicsFactory;
> >  import com.google.common.collect.ImmutableList;
> >
> > -import org.codehaus.jackson.annotate.JsonIgnore;
> > +import com.fasterxml.jackson.annotation.JsonIgnore;
>
> Can more of this file go away now? I think Storage.Objects is still
> needed but the Genomics stuff is no longer in use anywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/55/files#r28556833>.
>